api/http.go

```diff
@@ -67,7 +67,8 @@ func NewCatalystAPIRouter(cli config.Cli, vodEngine *pipeline.Coordinator, bal b
 	router.GET("/ok", withLogging(catalystApiHandlers.Ok()))
 	if cli.EnableAnalytics == "true" || cli.EnableAnalytics == "enabled" {
 		analyticsApiHandlers := handlers.NewAnalyticsHandlersCollection(mapic, lapi, cli.AnalyticsMetricsURL, cli.NodeName)
-		router.POST("/analytics/log", withCORS(analyticsApiHandlers.Log()))
+		router.POST("/analytics/log", withLogging(withCORS(analyticsApiHandlers.Log())))
+		router.GET("/analytics/log", withLogging(withCORS(geoHandlers.RedirectHandler())))
```

Comment:
Could you extend [the tests](https://github.com/livepeer/catalyst-api/blob/main/handlers/analytics_test.go#L145-L162) so they check CORS headers and GET request as well?

Reply:
Hmm, CORS and redirect kind-of not belong to `analytics.go`, because they are added from outside (in `http.go`). I see that for other cases we don't test it here, only in cucumber: https://github.com/livepeer/catalyst-api/blob/10b16410706c875ff220b3dfe4e1a6524e3b05c4/test/features/playback.feature#L51

We could add something similar, but technically it's already tested, so probably to safe to leave it as it is.